### PR TITLE
prepare database

### PR DESCRIPTION
**Pull request by duccs-0007**
create Task model and migrate.
create relationship between Task and User.

### CODE REVIEW

**Comment by duccs-0007** [comment]
ready

File: app/Models/Task.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Task extends Model
+{
+    //
+    protected $fillable = ['name'];
```
Comment (tranducquoc) [inline app/Models/Task.php]: protected $fillable = [
'name',
];

**Comment by duccs-0007** [comment]
ready

**Comment by hhhhuy112** [comment]
ready

File: app/Models/Task.php [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+namespace App\Models;
+
+use App\Models\User;
+use Illuminate\Database\Eloquent\Model;
+
+class Task extends Model
+{
+    //
+    protected $fillable = [
+    'name',
```
Comment (tranducquoc) [inline app/Models/Task.php]: thụt vào 4 space em

**Comment by duccs-0007** [comment]
ready

**Comment by chien-pv** [comment]
merged

**Pull request merged**